File: testdata/bench/lua/BenchmarkTest00002.lua
local pgmoon = require "pgmoon"
local args = ngx.req.get_uri_args()
local name = args.name
local pg = pgmoon.new({host="127.0.0.1", database="app"})
pg:connect()
local res = pg:query("SELECT * FROM users WHERE name = $1", name)
ngx.say(res)

File: testdata/bench/perl/BenchmarkTest00002.pl
use CGI;
my $cgi = CGI->new;
my $name = $cgi->param('name');
my $dbh = DBI->connect("dbi:mysql:testdb", "root", "");
my $sth = $dbh->prepare("SELECT * FROM users WHERE name = ?");
$sth->execute($name);
